tests/unit/algorithms/classification/adapters/openvino/test_openvino_models.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+import numpy as np
+
+from openvino.model_zoo.model_api.adapters import create_core
+from otx.algorithms.classification.tasks.openvino import OpenvinoAdapter
+from openvino.model_zoo.model_api.models.classification import Classification
+from otx.algorithms.classification.adapters.openvino.model_wrappers import (
+    OTXClassification,
+)
+from tests.test_suite.e2e_test_system import e2e_pytest_unit
+from tests.unit.api.parameters_validation.validation_helper import (
+    check_value_error_exception_raised,
+)
+from tests.unit.algorithms.classification.test_helper import (
+    generate_label_schema,
+    generate_cls_dataset
+)
+from otx.algorithms.classification.utils.labels_utils import get_multihead_class_info
+
+
+class MockClassification(OTXClassification, Classification):
+    def __init__(self):
+        self.out_layer_name = 'logits'
+        hierarchical_dataset = generate_cls_dataset(hierarchical=True)
+        label_schema = generate_label_schema(hierarchical_dataset.get_labels(),
+                                         multilabel=False,
+                                         hierarchical=True)
+        self.multihead_class_info = get_multihead_class_info(label_schema)
+
+
+class TestOTXClassification:
+    @pytest.fixture(autouse=True)
+    def setup(self, mocker) -> None:
```

Comment:
Could you remove `mocker` argument if the function doesn't use it? It would be better to modify not only this but also some other functions same as this function.